src/Bonsai.Lsl/StreamInlet.cs

```diff
@@ -57,6 +57,12 @@ public class StreamInlet : ZeroArgumentExpressionBuilder, INamedElement
         [Description("Specifies the data format of each channel in the LSL stream.")]
         public ChannelFormat ChannelFormat { get; set; } = ChannelFormat.Float32;
 
+        /// <summary>
+        /// Gets or sets a value specifying the postprocessing time-synchronisation options to use on an LSL stream inlet.
+        /// </summary>
+        [Description("Specifies the postprocessing options of an LSL stream inlet.")]
```

Comment:
For consistency, prefer US-english spelling in code and code comments (e.g. synchronization). Would update both descriptions for styling consistency with other properties and to match the XML comment (also there seems to be no hyphen in time-synchronization from looking at LSL docs):

Gets or sets a value specifying the postprocessing time synchronization options to use on the LSL stream.

"Specifies the postprocessing time synchronization options to use on the LSL stream."

Reply:
Changed in b7a21e20b9cc13eb9ade43d3157271b65ac68caa